traffic_ops/install/lib/GenerateCert.pm

```diff
@@ -70,13 +70,9 @@ sub writeCdn_conf {
         };
     }
 
-    # dump conf data in compact but readable form
-    my $dumper = Data::Dumper->new( [$cdnh] );
-    $dumper->Indent(1)->Terse(1)->Quotekeys(0);
-
     # write whole config to temp file in pwd (keeps in same filesystem)
     my $tmpfile = File::Temp->new( DIR => '.' );
-    print $tmpfile $dumper->Dump();
+    writeJson("$tmpfile", $cdnh);
```

Comment:
putting a variable in quotes like this is necessary in sh -- not in perl..

Other than that,  this looks good..   change that and I'll merge..